packages/firestore/src/local/indexeddb_mutation_queue.ts

```diff
@@ -139,7 +120,7 @@ export class IndexedDbMutationQueue implements MutationQueue {
   static loadNextBatchIdFromDb(
     txn: PersistenceTransaction
   ): PersistencePromise<BatchId> {
-    let maxBatchId = BATCHID_UNKNOWN;
+    let maxBatchId = 0;
```

Comment:
Hrm...  Why did this change?  You want the first batchId to be 1 instead of 0 for some reason?